src/resmom/stage_func.c

```diff
@@ -1465,6 +1492,8 @@ sys_copy(int dir, int rmtflg, char *owner, char *src, struct rqfpair *pair, int
 
```

Comment:
[https://github.com/PBSPro/pbspro/blob/833f6196423d5b751d8416ad2bbe4927bb058c0e/src/resmom/stage_func.c#L1488](url)
Error handling for getpwd.